File: Python/1768.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        wordlen = len(word1)
        if len(word2)>wordlen:
            wordlen = len(word2)
        
        merge_word = ''
        for i in range(0, wordlen):
            if i < len(word1):
                merge_word += word1[i]
            if i < len(word2):
                merge_word += word2[i]
        
        return merge_word

File: C++/1768.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int maxlen = word1.length() > word2.length() ? word1.length() : word2.length();
        string merged_word = "";
        for(int i = 0 ; i < maxlen ; ++i)
        {
            if(i < word1.length()) merged_word += word1[i];
            if(i < word2.length()) merged_word += word2[i];
        }
        return merged_word;
    }
};